snippets/survival_analysis/KaplanMeier_curves/KaplanMeier.Rmd

```diff
@@ -1,83 +1,101 @@
 ---
-title: "KaplanMeier"
+title: "Kaplan Meier"
 output: html_document
 ---
 
 ```{r setup, include=FALSE}
 knitr::opts_chunk$set(echo = TRUE)
 ```
 
-# Read in data
+# Read in survival data
 ```{r}
 library(data.table)
+library(survival)
+library(RColorBrewer)
 
-surv_dat = fread('/home/jana/Github/bbsnippets/data/survival_example_data.csv',header = T)
+surv_dat = fread('~/Github/bbsnippets/data/survival_example_data.csv',header = T)
```

Comment:
And consider switching to `TRUE` instead of `T` for the official examples ;)

Reply:
Fine ;)